viewer/db.js

```diff
@@ -154,10 +155,17 @@ Db.initialize = async (info, cb) => {
     console.log(`prefix:${internals.prefix} usersPrefix:${internals.usersPrefix}`);
   }
 
+  // build regular expressions for the user-specified extra query index patterns
+  if (Array.isArray(info.queryExtraIndices)) {
+    for (const pattern in info.queryExtraIndices) {
+      internals.queryExtraIndicesRegex.push(ArkimeUtil.wildcardToRegexp(info.queryExtraIndices[pattern]));
+    }
+  }
+
   // Update aliases cache so -shrink/-reindex works
   if (internals.nodeName !== undefined) {
-    Db.getAliasesCache(['sessions2-*', 'sessions3-*']);
-    setInterval(() => { Db.getAliasesCache(['sessions2-*', 'sessions3-*']); }, 2 * 60 * 1000);
+    Db.getAliasesCache(Db.defaultIndexPatterns(info.queryExtraIndices));
+    setInterval(() => { Db.getAliasesCache(Db.defaultIndexPatterns(info.queryExtraIndices)); }, 2 * 60 * 1000);
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array (pulled from the config's `queryExtraIndices` value) and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.